git-rundown.sh

```diff
@@ -37,31 +63,31 @@ do
 		echo -en "\033[0;36m${s:10}\033[0m "
 
 		# Check for modified files
-		if [ $(git status | grep modified -c) -ne 0 ]
+		if [ $(git status | grep $check_modif -c) -ne 0 ]
```

Comment:
Each variable needs double quotes throughout the script:

```suggestion
		if [ $(git status | grep "$check_modif" -c) -ne 0 ]
```
```